src/github.com/mozilla-services/pushgo/simplepush/net.go

```diff
@@ -126,12 +153,14 @@ type ServeCloser struct {
 	closeOnce Once
 }
 
-// Close stops the server.
+// Close stops the server. The returned error is always nil; it is included
+// to satisfy the io.Closer interface.
 func (s *ServeCloser) Close() error {
 	return s.closeOnce.Do(s.close)
 }
 
 func (s *ServeCloser) close() error {
+	s.SetKeepAlivesEnabled(false)
```

Comment:
It's a sanity check in case the server accepts a connection as it's shutting down—no need to use HTTP keep-alive for a connection that's about to be closed, anyway.